quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/projects/gradle/quarkus-gradle-project/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
don't need to keep that file in git

Reply:
Seems like these `.classpath` and `.project` files are added because of the .gitignore I copied over from com.redhat.microprofile.test which overwrites the root .gitignore to tell git to keep them around. (https://github.com/redhat-developer/quarkus-ls/blob/master/microprofile.jdt/com.redhat.microprofile.jdt.test/projects/maven/.gitignore, https://github.com/redhat-developer/quarkus-ls/blob/master/microprofile.jdt/com.redhat.microprofile.jdt.test/projects/gradle/.gitignore)

Any reason for keeping these .classpath/.project files? Or is it it safe to remove them and remove the .gitignore that keeps them around @angelozerr @fbricon

Reply:
oh that might be caused by the tests importing the project directly as eclipse projects, not doing the regular maven or gradle project import.
In that case we should keep them

Reply:
Indeed, we could improve that in an another PR, but today we need .project and .classpath. I don't know if import is easy to implement with test.